inttest/common/footloosesuite.go

```diff
@@ -286,9 +286,9 @@ func (s *FootlooseSuite) RunWorkers(dataDir string) error {
 
 	var workerCommand string
 	if dataDir != "" {
-		workerCommand = fmt.Sprintf(`nohup k0s --debug --data-dir=%s worker --labels="k0sproject.io/foo=bar" "%s" - >/tmp/k0s-worker.log 2>&1 &`, dataDir, token)
+		workerCommand = fmt.Sprintf(`nohup k0s --debug --data-dir=%s worker --labels="k0sproject.io/foo=bar" --kubelet-extra-args="--address=0.0.0.0" "%s" - >/tmp/k0s-worker.log 2>&1 &`, dataDir, token)
 	} else {
-		workerCommand = fmt.Sprintf(`nohup k0s --debug worker --labels="k0sproject.io/foo=bar" "%s" >/tmp/k0s-worker.log 2>&1 &`, token)
+		workerCommand = fmt.Sprintf(`nohup k0s --debug worker --labels="k0sproject.io/foo=bar" --kubelet-extra-args="--address=0.0.0.0" "%s" >/tmp/k0s-worker.log 2>&1 &`, token)
```

Comment:
I think we should refactor `RunWorkers` to take a list of args instead of adding those args all tests.

Similar to what we did in df25442e1479e02dc59bfce62ac84f10befd7294

Can/should be done as separate issue though.

Reply:
Yup, shall be refactored. I'll actually do it in this PR, better to do it sooner rather than later.

Just wanted to push early for general "design level" discussions plus to get the smokes running early

Reply:
@ncopa I refactored this to be more flexible, PTAL